mage/main.go

```diff
@@ -401,3 +439,22 @@ func RunCompiled(inv Invocation, exePath string) int {
 	}
 	return sh.ExitStatus(c.Run())
 }
+
+func removeContents(dir string) error {
+	d, err := os.Open(dir)
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+	names, err := d.Readdirnames(-1)
+	if err != nil {
+		return err
+	}
+	for _, name := range names {
+		err = os.RemoveAll(filepath.Join(dir, name))
```

Comment:
if you use ReadDir, then you could check if something was a file before removing it, that way if someone happens to make a subdirectory in this directory, we won't wipe it out.